website/src/plugins/remark-page-title.ts

```
@@ -0,0 +1,37 @@
+import type { Root } from 'mdast';
+import { toString } from 'mdast-util-to-string';
```

ESLint

[error] "mdast-util-to-string" is extraneous.

(n/no-extraneous-import)